agot-bg-game-server/src/common/ingame-game-state/game-data-structure/house-card/TywinLannisterHouseCardAbility.ts

```diff
@@ -1,5 +1,23 @@
 import HouseCardAbility from "./HouseCardAbility";
+import AfterWinnerDeterminationGameState
+    from "../../action-game-state/resolve-march-order-game-state/combat-game-state/post-combat-game-state/after-winner-determination-game-state/AfterWinnerDeterminationGameState";
+import House from "../House";
+import HouseCard from "./HouseCard";
 
 export default class TywinLannisterHouseCardAbility extends HouseCardAbility {
```

Comment:
The file `TywinLannisterHouseCardAbility` was already created, weirdly. This PR then just fills the class.